scripts/release_scripts/cut_release_or_hotfix_branch.py

```diff
@@ -49,30 +49,33 @@
 from scripts import common
 
 
-def new_version_type(arg, pattern=re.compile(r'\d\.\d\.\d')):
-    """Checks that the new version name matches the expected pattern.
+def require_release_version_to_have_correct_format(
+        arg, pattern=re.compile(r'\d\.\d\.\d')):
+    """Checks that the release version name matches the expected pattern.
 
     Args:
-        arg: str. The new version name.
+        arg: str. The release version name.
         pattern: RegularExpression. The pattern that release version should
             match.
 
     Raises:
-        argparse.ArgumentTypeError: The new version name does not match
+        argparse.ArgumentTypeError: The release version name does not match
             the pattern.
 
     Returns:
-        str. The new version name with correct pattern.
+        str. The release version name with correct pattern.
     """
     if not pattern.match(arg):
         raise argparse.ArgumentTypeError(
-            'The format of "new_version" should be: x.x.x')
+            'The format of "release_version" should be: x.x.x')
     return arg
 
 
 _PARSER = argparse.ArgumentParser()
 _PARSER.add_argument(
-    '--new_version', help='new version to be released', type=new_version_type)
+    '--release_version',
+    help='version of the release for which the hotfix is being created',
```

Comment:
The file name says this script can be used for release or hotfix branch cut, Can we add a message which will indicate that this flag will be used for both purposes?

Reply:
Done.